layouts/partials/events/render-event.html

```diff
@@ -57,6 +57,7 @@ <h2>Content</h2>
 
 {{ $anchor_base := anchorize $event_name }}
 {{ $additional_types := partial "json-schema/resolve-additional-types" (dict "schema" $event_data.properties.content "anchor_base" $anchor_base) }}
+{{ $additional_types = uniq $additional_types }}
```

Comment:
I do wonder if we should move this into `resolve-additional-types` rather than doing it everywhere it gets called, but I guess this is fine